runelite-client/src/main/java/net/runelite/client/chat/ChatMessageManager.java

```diff
@@ -196,7 +196,8 @@ else if (client.isFriended(sanitizedUsername, true))
 			// Replace </col> tags in the message with the new color so embedded </col> won't reset the color
 			final Color color = chatColor.getColor();
 			message = ColorUtil.wrapWithColorTag(
-				message.replace(ColorUtil.CLOSING_COLOR_TAG, ColorUtil.colorTag(color)),
+				message.replace(ColorUtil.CLOSING_COLOR_TAG, ColorUtil.colorTag(color))
+					.replaceAll("<br>", ColorUtil.CLOSING_COLOR_TAG + "<br>" + ColorUtil.colorTag(color)),
```

Comment:
Is the closing color tag needed? Based on the behavior demonstrated in the provided screenshot, this should suffice.
```suggestion
					.replaceAll("<br>", "<br>" + ColorUtil.colorTag(color)),
```